spec/features/ncr/work_orders/approve_spec.rb

```diff
@@ -21,6 +21,21 @@
     end
   end
 
+  scenario "allows delegate to approve work order" do
+    Timecop.freeze do
```

Comment:
not sure. cargo-culted that layout from the test above it. The `begin` block sets the time 10 hours in the past so presumably there is some time flow control going on. My guess would be timezone-related checks, since I waded through that mess a few months ago.